plasma_framework/contracts/src/exits/payment/routers/PaymentInFlightExitRouterArgs.sol

```diff
@@ -18,4 +18,26 @@ library PaymentInFlightExitRouterArgs {
         bytes[] inputTxsInclusionProofs;
         bytes[] inFlightTxWitnesses;
     }
+
+    /**
+     * @notice Wraps arguments for challenge in-flight exit not canonical.
+     * @param inFlightTx RLP encoded in-flight transaction.
+     * @param inFlightTxInputIndex Index of shared input in transaction in flight.
+     * @param competingTx RLP encoded competing transaction.
+     * @param competingTxInputIndex Index of shared input in competing transaction.
+     * @param competingTxInputOutputType Output type of shared input.
+     * @param competingTxPos (optional) Position of competing transaction in chain if included.
+     * @param competingTxInclusionProof (optional) Merkle proofs that show the competing transaction was contained in chain.
+     * @param competingTxWitness Witness for competing transaction.
+     */
+    struct ChallengeCanonicityArgs {
+        bytes inFlightTx;
+        uint8 inFlightTxInputIndex;
```

Comment:
Tried to change to `uint16` as [suggested](https://github.com/omisego/plasma-contracts/pull/210#discussion_r316495119), it seems simple change causes many issues including SpendingCondition contract failed to deploy... 
Can we address it in subsequest PR - this one stuck for too long...

Reply:
sure.

Reply:
This should be resolved as #214 efforts so I won't created another issue for `index` type change `uint8 -> uint16`